apps/dashboard/src/components/variable/use-variable-list-keyboard-navigation.ts

```diff
@@ -0,0 +1,73 @@
+import { useRef } from 'react';
+
+import { useState } from 'react';
+
+export const useVariableListKeyboardNavigation = ({ maxIndex }: { maxIndex: number }) => {
```

Comment:
I'd move the hook inside the component, the file is not that big in total, <200 lines.